contracts/cw1155-base/src/contract.rs

```diff
@@ -207,18 +201,16 @@ pub fn execute_mint(
     let event = execute_transfer_inner(&mut deps, None, Some(&to), &token_id, amount)?;
     event.add_attributes(&mut rsp);
 
-    rsp.messages = if let Some(msg) = msg {
-        vec![Cw1155ReceiveMsg {
+    if let Some(msg) = msg {
+        rsp.messages = vec![Cw1155ReceiveMsg {
```

Comment:
This could also be `rsp.add_message(Cw1155ReceiveMsg { .. }.into_cosmos_msg(to)?);`

But I don't think that is any clearer in this case.